pkg/oauth/claims.go

```diff
@@ -0,0 +1,39 @@
+package oauth
+
+import (
+	"context"
+	"time"
+)
+
+// TokenClaims represents extracted token claims
+type TokenClaims struct {
```

Comment:
Is there no lib for this? 

No big deal, but sounds super generic

Reply:
As lib we're using lestrrat-go/jwx. Unfortunately the have the Token behind an interface (and their implementation is private), so we can't reuse it :/
But let me remove the json tags, because this was for testing and not needed anymore IIRC